### PR TITLE
Swapping the order of episodes on lists and loops

### PR DESCRIPTION
**Pull request by maxim-belkin**
See #496 for more info.

### CODE REVIEW

File: _episodes/05-loop.md [hunk 1]
```diff
@@ -191,65 +193,66 @@ Here's another loop that repeatedly updates a variable:
...
 The first time around,
 `length` is zero (the value assigned to it on line 1)
-and `vowel` is `'a'`.
+and `value` is `Curie`.
 The statement adds 1 to the old value of `length`,
 producing 1,
 and updates `length` to refer to that new value.
 The next time around,
-`vowel` is `'e'` and `length` is 1,
++`value` is `Darwin` and `length` is 1,
```
Comment (ldko) [inline _episodes/05-loop.md:218]: typo
```suggestion
`value` is `Darwin` and `length` is 1,
```

File: _episodes/05-loop.md [hunk 2]
```diff
@@ -191,65 +193,66 @@ Here's another loop that repeatedly updates a variable:
...
-`length` is 5;
-since there is nothing left in `'aeiou'` for Python to process,
+After one more update,
+`length` is 3;
+since there is nothing left in `names` for Python to process,
 the loop finishes
-and the `print` statement on line 4 tells us our final answer.
+and the `print` function on line 5 tells us our final answer.
 
 Note that a loop variable is a variable that's being used to record progress in a loop.
```
Comment (ldko) [inline _episodes/05-loop.md:226]:
```suggestion
Note that a loop variable is a variable that is being used to record progress in a loop.
```

**Review by ldko — COMMENTED** [review]
I think the swap looks good. I noticed a few minor things that we might want to change.

File: _episodes/05-loop.md [hunk 3]
```diff
@@ -24,114 +24,114 @@ We have a dozen data sets right now, though, and more on the way.
...
 ~~~
 {: .language-python}
 
-In Python, a string is basically an ordered collection of characters, and every
-character has a unique number associated with it -- its index. This means that
-we can access characters in a string using their indices.
-For example, we can get the first character of the word `'lead'`, by using
-`word[0]`. One way to print each character is to use four `print` statements:
+In Python, a list is basically an ordered collection of elements, and every
+element has a unique number associated with it -- its index. This means that
```
Comment (ldko) [inline _episodes/05-loop.md:37]:
```suggestion
element has a unique number associated with it --- its index. This means that
```

**Comment by maxim-belkin** [comment]
Thank you, Lauren. All good suggestions/edits.

**Review by ldko — APPROVED** [review]
This looks good to me. I see the there is a build failure, but it doesn't look specifically relevant to the changes here. `make lesson-check-all` outputs what I would expect locally. 

Thanks very much for wrapping this up @maxim-belkin ! I imagine a lot of people will be glad to have this change in the lesson. Thanks to everyone who worked on it!

**Review by annefou — APPROVED** [review]
This is excellent! I also don't know why build fails: it works well for me locally.

I am really looking forward to be teaching this "new" lesson!!! thanks @maxim-belkin

**Comment by maxim-belkin** [comment]
The build fails because of the missing `github-pages` gem in Travis. I guess the fix is to add `github-pages` to the list of Gems we install on line 10 in `.travis.yml` https://github.com/swcarpentry/python-novice-inflammation/blob/4f31efad053decd7f08c39b41d14a859ad1cc344/.travis.yml#L10
but that's not critical because the main test -- GitHub Action "Website" -- passes.

As we discussed previously, we need to announce this change. Do you think we should do it before or after merging this PR?

**Comment by ldko** [comment]
> As we discussed previously, we need to announce this change. Do you think we should do it before or after merging this PR?

On the issue [you mention](https://github.com/swcarpentry/python-novice-inflammation/issues/496#issuecomment-814606963) announcing it on the Carpentries' Discuss channel. I am not sure what that refers to :D. Will a lot of people see it there? It does make sense to give notice though, in particular because of the accounts mentioned in this [recent RFC](https://github.com/carpentries/maintainer-RFCs/issues/16). It seems to me not waiting too long to merge it though would be good, so we don't lose hold of it again. Does it also make sense to do a tag/release for the version before we merge the change?

**Comment by annefou** [comment]
It is a good idea to do a tag/release for the version before merging. For announcing it: I am not sure what is best but announcing it in the Carpentries' Discuss channel seems a good idea (I guess many people will be reached).

**Comment by maxim-belkin** [comment]
> Does it also make sense to do a tag/release for the version before we merge the change?  

My understanding is that lesson releases are coordinated across The Carpentries org(s) and are performed by The Carpentries' staff.

@fmichonneau, what are our options here?

**Comment by maxim-belkin** [comment]
> On the issue [you mention](https://github.com/swcarpentry/python-novice-inflammation/issues/496#issuecomment-814606963) announcing it on the Carpentries' Discuss channel. I am not sure what that refers to :D

I was talking abouthttps://carpentries.topicbox.com/ . For example, this is our previous Request (Call) for Contributions: https://carpentries.topicbox.com/groups/discuss/Tb9e8878a4c4c4fa8-M281b5e6b472544bacf432926/call-for-contributions-visuals-images-figures-plots-charts-etc

**Pull request merged**